build.sbt

```diff
@@ -499,65 +486,37 @@ lazy val mergeSettings = Def.settings(
   }
 )
 
-// Pre-release versions go to bintray and should be published via `publish`.
-// This is the default behavior that you get without modifying the build.
-// The only exception is that we take extra care to not publish on pull request validation jobs in Drone.
-def shouldPublishToBintray: Boolean = {
-  if (!new File(sys.props("user.home") + "/.bintray/.credentials").exists) return false
-  if (sys.props("sbt.prohibit.publish") != null) return false
-  if (sys.env.contains("CI_PULL_REQUEST")) return false
-  LibraryVersion.contains("-")
-}
-
-// Release versions go to sonatype and then get synced to maven central.
-// They should be published via `publish-signed` and signed by someone from <developers>.
-// In order to publish a release version, change LibraryVersion to be equal to LibrarySeries.
-def shouldPublishToSonatype: Boolean = {
-  if (!os.secret.obtain("sonatype").isDefined) return false
-  if (sys.props("sbt.prohibit.publish") != null) return false
-  !LibraryVersion.contains("-")
-}
-
 lazy val publishableSettings = Def.settings(
   sharedSettings,
   bintrayOrganization := Some("scalameta"),
+  bintrayRepository := (if (LibraryVersion.contains("-")) "pre-release" else "stable"),
```

Comment:
Not necessary right now, but I think we should consider removing the custom version scheme in favor to sbt-dynver https://github.com/dwijnand/sbt-dynver

The `+` suffix follows semantic versioning rule 10 about attaching metadata to version numbers http://semver.org/spec/v2.0.0.html